glide.lock

```diff
@@ -1,12 +1,14 @@
 hash: e8d332f58ad0a43b3d06d428ac2df127cbbff2042b3bfffa5e94cf90d0fd36b0
-updated: 2016-12-07T13:40:09.6559911-05:00
+updated: 2016-12-16T00:08:25.500002655+05:30
 imports:
 - name: github.com/boltdb/bolt
   version: 583e8937c61f1af6513608ccc75c97b6abdf4ff9
 - name: github.com/btcsuite/btclog
   version: 73889fb79bd687870312b6e40effcecffbd57d30
 - name: github.com/btcsuite/fastsha256
   version: 637e656429416087660c84436a2a035d69d54e2e
+- name: github.com/btcsuite/go-flags
```

Comment:
I don't see addition of this to the glide.yaml.. but anyways this shouldn't be added.  We switched to upstream repo pkg path.  You can see `github.com/jessevdk/go-flags` further down in the lockfile.  Please change all uses of btcsuite/go-flags to jessevdk/go-flags to avoid pulling the same package twice under different names.